docs/development_suite/api-console/api-design/microservice-containers.md

```diff
@@ -0,0 +1,89 @@
+---
+id: microservice-containers
+title: Microservice Containers
+sidebar_label: Microservice Containers
+---
+
+# Microservice Containers
+
+A microservice in Console is a collection of one or more containers that solve a single yet well defined business scope. 
+
+In general microservices define a single docker image and thus a single **main container**.
+
+Sometimes tough, it may be useful to run another container alongside the main container to add some capabilities. Usually these kind of containers are used to avoid cluttering the main service with helper code. These containers are known as **sidecar containers**.
```

Comment:
Another small typo

```suggestion
Sometimes though, it may be useful to run another container alongside the main container to add some capabilities. Usually these kind of containers are used to avoid cluttering the main service with helper code. These containers are known as **sidecar containers**.
```